parsec/mca/device/device_gpu.c

```diff
@@ -702,8 +701,8 @@ parsec_device_memory_reserve( parsec_device_gpu_module_t* gpu_device,
         mem_elem_per_gpu = alloc_size / eltsize;
         gpu_device->memory = zone_malloc_init( base_ptr, mem_elem_per_gpu, eltsize );
         if( gpu_device->memory == NULL ) {
-            parsec_warning("GPU[%d:%s] Cannot allocate memory on GPU %s. Skip it!",
-                           gpu_device->super.device_index, gpu_device->super.name, gpu_device->super.name);
+            parsec_warning("GPU[%d:%s] Failed trying to allocate %zu bytes. We tried to do so based on an initial_free_mem of %zu bytes and elt_size of %zu bytes",
+                           gpu_device->super.device_index, gpu_device->super.name, gpu_device->super.name, initial_free_mem, elt_size);
```

Comment:
What @abouteiller said, plus:

```suggestion
                           gpu_device->super.device_index, gpu_device->super.name, gpu_device->super.name, initial_free_mem, eltsize);
```